locale/Makefile

```diff
@@ -6,13 +6,14 @@
 # make tx-update - download and merge translations from Transifex
 # make clean - clean everything
 #
-DOMAIN = hammer_cli_foreman_bootdisk
+DOMAIN = hammer-cli-foreman-bootdisk
```

Comment:
This isn't right, the domain configured in HammerCLIForemanBootdisk::I18n uses underscores.